--- konwert-1.8.orig/bin/trs.cc	Tue Sep  1 12:24:52 1998
+++ konwert-1.8/bin/trs.cc	Sun Mar 24 13:23:44 2002
@@ -7,13 +7,15 @@
 //     \__/              qrczak@knm.org.pl
 //      ^^      http://qrczak.home.ml.org/
 
-#include <string.h>
-#include <iostream.h>
-#include <fstream.h>
-#include <strstream.h>
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <strstream>
+#include <stdlib.h>
 #include <ctype.h>
 #include <unistd.h>
 #define _(String) (String)
+using namespace std;
 
 #define WERSJA "1.8"
 #define WLKSLOWA 64*1024	// Maksymalna d³ugo¶æ s³owa do zamiany
@@ -133,7 +135,7 @@
 
 struct zmienna
 {
-	int zmienna, wartosc;
+	int nazwa, wartosc;
 	zmienna *nast;
 };
 
@@ -315,7 +317,7 @@
 					)
 					{
 						*zmienne = new zmienna;
-						(*zmienne)->zmienna =
+						(*zmienne)->nazwa =
 							s1[0] >= 'A' && s1[0] <= 'Z' ?
 								s1[0] - 'A'
 							:
@@ -542,7 +544,7 @@
 					zmienna *z = r->warunki;
 					while (z)
 					{
-						if (zmienne[z->zmienna] != z->wartosc)
+						if (zmienne[z->nazwa] != z->wartosc)
 							goto niepasuje;
 						z = z->nast;
 					}
@@ -572,7 +574,7 @@
 					z = r->zmienne;
 					while (z)
 					{
-						zmienne[z->zmienna] = z->wartosc;
+						zmienne[z->nazwa] = z->wartosc;
 						z = z->nast;
 					}
 					goto zamienione;
@@ -588,7 +590,7 @@
 
 /******** PROGRAM G£ÓWNY *****************************************************/
 
-main (int argc, char **argv)
+int main (int argc, char **argv)
 {
 /*
 	setlocale (LC_ALL, "");
